### PR TITLE
win_virtio_driver_update: Correct the logic

### PR DESCRIPTION
**Pull request by luckyh**
Correct the logic to avoid installing driver twice.

Signed-off-by: Xu Han <xuhan@redhat.com>

ID: 1656668

### CODE REVIEW

File: qemu/tests/win_virtio_driver_update_test.py [hunk 1]
```diff
@@ -39,13 +39,7 @@ def change_virtio_media(cdrom_virtio):
...
-        error_context.context("Uninstall virtio driver", logging.info)
-        single_driver_install.run(test, params, env)
-        # Need install driver after uninstallation.
-        params["need_uninstall"] = False
-        error_context.context("Install virtio driver", logging.info)
-    else:
+    if params.get("need_uninstall") != "yes":
         error_context.context("Downgrade virtio driver", logging.info)
         change_virtio_media(params["cdrom_virtio_downgrade"])
         single_driver_install.run(test, params, env)
```
Comment (lijinlijin) [inline qemu/tests/win_virtio_driver_update_test.py]: A minor issue: maybe we should move  line 52:change_virtio_media(params["cdrom_virtio"]) to if part as well.

**Comment by lijinlijin** [comment]
Others look good to me:
 (1/3) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (1846.44 s)
 (2/3) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.during_bg_test.uninstall_install: PASS (375.35 s)
 (3/3) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.win_virtio_driver_update_test.with_balloon.during_bg_test.downgrade_upgrade: PASS (609.07 s)

File: qemu/tests/win_virtio_driver_update_test.py (hunk 1 above)
Comment (luckyh) [reply qemu/tests/win_virtio_driver_update_test.py]: Ah, right, will update it as will, thanks for your review.

**Comment by lijinlijin** [comment]
Acked-by: Li Jin lijin@redhat.com

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**